diff --git a/src/app/Config.ts b/src/app/Config.ts
index 081a00a..194b66a 100644
--- a/src/app/Config.ts
+++ b/src/app/Config.ts
@@ -143,7 +143,7 @@ const Config = {
 	TileServerEndpoint: 'https://tiles.streets.gl',
 	SlippyEndpointTemplate: 'https://tile.openstreetmap.org/{z}/{x}/{y}.png',
 	ElevationEndpointTemplate: 'https://tiles.streets.gl/elevation/{z}/{x}/{y}',
-	TilesEndpointTemplate: 'https://tiles.streets.gl/vector/{z}/{x}/{y}'
+	TilesEndpointTemplate: 'http://localhost:8081/data/data/{z}/{x}/{y}.pbf'
 };
 
 export default Config;
diff --git a/webpack.config.js b/webpack.config.js
index 82590b3..be43fb2 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -6,10 +6,9 @@ const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const ESLintPlugin = require('eslint-webpack-plugin');
 const {EsbuildPlugin} = require('esbuild-loader');
 
-const childProcess = require('child_process');
 const {DefinePlugin} = require("webpack");
-const COMMIT_SHA = childProcess.execSync('git rev-parse HEAD').toString().trim();
-const COMMIT_BRANCH = childProcess.execSync("git rev-parse --abbrev-ref HEAD").toString().trim();
+const COMMIT_SHA = 'hello';
+const COMMIT_BRANCH = 'world';
 const VERSION = require('./package.json').version;
 
 module.exports = (env, argv) => ([{
